sys/sys/atomic_common.h

```diff
@@ -45,7 +45,7 @@
 #define	atomic_load_8(p)	(*(volatile uint8_t *)(p))
 #define	atomic_load_16(p)	(*(volatile uint16_t *)(p))
 #define	atomic_load_32(p)	(*(volatile uint32_t *)(p))
-#ifdef _LP64
```

Comment:
I should really update the compiler to poison `_LP64/__LP64__` in purecap mode, but last time I had a look that also reported errors even if the identifier was not actually being used.

Reply:
https://cheri-compiler-explorer.cl.cam.ac.uk/z/1Panf4

Reply:
That would be nice, I suspect there are still a bunch lurking around.